src/st/class-wpml-pb-integration.php

```diff
@@ -314,6 +319,14 @@ public function get_factory() {
 		return $this->factory;
 	}
 
+	public function get_string_clean_up( $_, $postId ) {
```

Comment:
This looks like a leftover @brucepearson.

Reply:
@strategio why? It's hooked above: `add_filter( 'wpml_pb_get_string_clean_up', [ $this, 'get_string_clean_up' ], 10, 2 );`

Reply:
Ah right @andreasciamanna, my bad :)